### PR TITLE
Adding sample config for ForgeRock as IdP

### PR DESCRIPTION
**Pull request by nervda**
Added sample config for ForgeRock, uploaded three screenshots and added links to new document in sidebar and general IdP config page

### CODE REVIEW

File: content/source/docs/enterprise/saml/identity-provider-configuration-forgerock.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+page_title: "SAML ForgeRock Identity Provider Configuration - Terraform Enterprise"
+---
+
+# ForgeRock Configuration
+
+Follow these steps to configure ForgeRock as the identity provider (IdP) for Terraform Enterprise.
+
+## Configure a new Remote Service Provider in ForgeRock
+
+On the TFE console, paste the ForgeRock Access Manager hosted IDP metadata certificate in the proper pem encoded format.  Enter the IDP SSO and SLO URLs as well.
```
Comment (nfagerlund) [inline content/source/docs/enterprise/saml/identity-provider-configuration-forgerock.md]: Instructions need to explain where to obtain these items. (In most of the other IdPs, these seem to be provided at the end of configuring the application.)

File: content/source/docs/enterprise/saml/identity-provider-configuration-forgerock.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+
+In the hosted IDP assertion content tab the highlighted entry must be on the top to be first in the order of precedence.
```
Comment (nfagerlund) [inline content/source/docs/enterprise/saml/identity-provider-configuration-forgerock.md]: Need to name the desired entry in the text of the instructions. Can't just screenshot it, that's not accessible.

File: content/source/docs/enterprise/saml/identity-provider-configuration-forgerock.md [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+
+![Screenshot: ForgeRock dialog for nameID Value map](./images/sso-forgerock-nameid.png)
+
+After a Remote SP is created in ForgeRock Access Manager with the provided TFE metadata.
```
Comment (nfagerlund) [inline content/source/docs/enterprise/saml/identity-provider-configuration-forgerock.md]: How?

File: content/source/docs/enterprise/saml/identity-provider-configuration-forgerock.md [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+
+The Assertion content tab of the Remote SP which was created should show as below.
```
Comment (nfagerlund) [inline content/source/docs/enterprise/saml/identity-provider-configuration-forgerock.md]: Describe, please!

**Comment by nfagerlund** [comment]
Ping! This pull request has become stale, and there are still changes necessary before we'd be willing to merge it. Do you think you'll be able to come back to this at some point? Is there any help you'd like in bringing this into a mergeable state?

**Comment by judithpatudith** [comment]
Hi! Since this has been sitting for a while, I'm going to go ahead and close it. @nervda if you get around to making the changes that Nick had requested, please feel free to re-open it! Thanks!

**Pull request closed without merging**